nginx/conf.d/include/locations.conf

```diff
@@ -11,6 +11,11 @@ location /webdav_health {
     access_log off;
 }
 
+location /redirect {
```

Comment:
Eventually we'll want three locations:
- `/redirect`
- `/proxy`
- `/cache`

and potentially a fourth default that chooses between the other three as appropriate for the path. What should that be called? `/get`?